Software/src/Settings.cpp

```diff
@@ -816,6 +816,8 @@ QStringList Settings::getSupportedSerialPortBaudRates()
     QStringList list;
 
     // TODO: Add more baud rates if need it
+    list.append("2000000"); 
+    list.append("1500000");
```

Comment:
Please remove this change from the PR. This change is about the number of LEDs, and apparently there are some rules about valid baud rates (#116). Feel free to create a separate PR for these, but it should include a UI change to indicate that higher rates are experimental.